README.md

```diff
@@ -24,14 +24,7 @@ where versionType corresponds to npm version types. This only works on non-Windo
 
 The following steps are needed before you can run migrations on a remote database in a private subnet of an AWS VPC.
 
-First, you will need to ensure that your current device IP address is whitelisted to use the EC2 instance.
-
-- Log into the AWS console and navigate to the EC2 Dashboard.
-- Navigate to the Security Groups tab by looking at the Network & Security section in the left pane and clicking on the Security Groups tab.
-- Look for the security group titled `cms-bastion-<ENVIRONMENT>`, where `<ENVIRONMENT>` is either staging or production. Click on the security group and then the Inbound rules tab.
```

Comment:
could we keep this but make it a sub-point so engineers know where this IP is specified?